src/app/modules/cadastre/cadastre.enums.ts

```diff
@@ -0,0 +1 @@
+export const CADASTRE_ORI = 'Cadastre_ori';
```

Comment:
Don't use the underscore notation for this value. Replace 'Cadastre_ori' with 'CadastreOri'.

Reply:
Just saw that it's alos in lib, where it should be. Remove this one but apply the previous comment to the lib version.